src/components/CheckboxToggle/CheckboxToggle.js

```diff
@@ -73,7 +73,10 @@ CheckboxToggle.propTypes = {
   /**
    * Checkbox label
    */
-  label: PropTypes.string.isRequired,
+  label: PropTypes.oneOfType([
+    PropTypes.string,
```

Comment:
If I am not mistaken `PropTypes.node.isRequired` can be used here directly, as it is compatible with strings also.
`PropTypes.node` - is the most generic type.

Reply:
Agreed, I'll change other occurrences of this pattern too.